digitalearthau/collections.py

```diff
@@ -384,47 +380,63 @@ def add_albers_collections(name: str, project='rs0'):
-                    f'/g/data/fk4/datacube/002/' + glob_offset,
-                    f'/g/data/fk4/datacube/.trash/' + glob_offset,
+                    '/g/data/{project}/datacube/002/FC/'
+                    'LS5_TM_{name}/*_*/LS5*{name}*.nc'.format(project=project,
+                                                              name=name.upper()),
                 ),
                 unique=('time.lower.day', 'lat', 'lon'),
+                index_=index,
                 # Tiles default to trusting index over the disk: they were indexed at the end of the job,
                 # so unfinished tiles could be left on disk.
                 trust=Trust.INDEX
-            )
-        )
-    # New FC
-    for sat in ['ls5', 'ls7', 'ls8']:
-        name = 'fc'
-        glob_offset = f'{sat.upper()}_TM_{name.upper()}/*_*/{sat.upper()}*{name.upper()}*.nc'
-        _add(
+            ),
             Collection(
-                name=f'{sat}_{name}_albers_staging',
-                query={'product': '{sat}_{name}_albers_staging'},
+                name='ls7_{}_albers'.format(name),
+                query={'product': 'ls7_{}_albers'.format(name)},
                 file_patterns=(
-                    f'/g/data/v10/public/data/fc/' + glob_offset,
-                    f'/g/data/fk4/datacube/002/' + glob_offset,
+                    '/g/data/{project}/datacube/002/FC/'
+                    'LS7_ETM_{name}/*_*/LS7*{name}*.nc'.format(project=project,
+                                                               name=name.upper()),
                 ),
                 unique=('time.lower.day', 'lat', 'lon'),
+                index_=index,
+                # Tiles default to trusting index over the disk: they were indexed at the end of the job,
+                # so unfinished tiles could be left on disk.
+                trust=Trust.INDEX
+            ),
+            Collection(
+                name='ls8_{}_albers'.format(name),
+                query={'product': 'ls8_{}_albers'.format(name)},
+                file_patterns=(
+                    '/g/data/{project}/datacube/002/FC/'
+                    'LS8_OLI_{name}/*_*/LS8*{name}*.nc'.format(project=project,
+                                                               name=name.upper()),
+                ),
+                unique=('time.lower.day', 'lat', 'lon'),
+                index_=index,
                 # Tiles default to trusting index over the disk: they were indexed at the end of the job,
                 # so unfinished tiles could be left on disk.
                 trust=Trust.INDEX
             )
         )
 
+    # PQ/NBAR/NBART Albers
+    add_albers_collections('pq')
+    add_albers_collections('nbar')
+    add_albers_collections('nbart')
+
+    # FC
+    # Example: ingested fractional cover:
+    # LS5_TM_FC  LS7_ETM_FC  LS8_OLI_FC
+    # /g/data/fk4/datacube/002/FC/LS5_TM_FC/13_-22/LS5_TM_FC_3577_13_-22_20030901235428500000_v1490733226.nc
+    add_fc_albers_collections('fc')
```

Comment:
If this function is only for `FC`, it would be simpler and nicer to not include the `name` argument.

Reply:
Updated the code by removing name argument.